### PR TITLE
TST: misplaced array tests

### PR DESCRIPTION
**Pull request by jbrockmendel**
- [ ] closes #xxxx (Replace xxxx with the Github issue number)
- [ ] [Tests added and passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#writing-tests) if fixing a bug or adding a new feature
- [ ] All [code checks passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit).
- [ ] Added an entry in the latest `doc/source/whatsnew/vX.X.X.rst` file if fixing a bug or adding a new feature.

### CODE REVIEW

File: pandas/tests/arrays/test_array.py [hunk 1]
```diff
@@ -391,61 +390,3 @@ def test_array_not_registered(registry_without_decimal):
...
-                pd.date_range("20120101", periods=10, freq="2D", tz="Asia/Hong_Kong"),
-                pd.Timestamp("20120102", tz="Asia/Hong_Kong"),
-            ],
-            [
-                pd.timedelta_range(start="1 day", end="10 days", periods=10),
-                pd.Timedelta("2 days"),
-            ],
-        ],
-    )
-    def test_search_sorted_datetime64_scalar(self, arr, val):
```
Comment (rhshadrach) [inline pandas/tests/arrays/test_array.py:441]: This get dropped?

Comment (jbrockmendel) [reply pandas/tests/arrays/test_array.py:441]: yes, redundant with existing tests in tests.arrays.test_datetimelike

**Review by rhshadrach — APPROVED** [review]
lgtm

**Review by mroeschke — APPROVED** [review]

**Comment by mroeschke** [comment]
Thanks @jbrockmendel

**Pull request merged**